src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.Common.targets

```diff
@@ -77,6 +77,7 @@ Copyright (c) .NET Foundation. All rights reserved.
 
   <PropertyGroup Condition="'$(RoslynCompilerType)' == 'FrameworkPackage' and '$(OS)' == 'Windows_NT' and '$(MSBuildRuntimeType)' == 'Full'">
     <RoslynTargetsPath>$(NuGetPackageRoot)\microsoft.net.sdk.compilers.toolset\$(NETCoreSdkVersion)</RoslynTargetsPath>
+    <RoslynTasksAssembly>$(RoslynTargetsPath)\Microsoft.Build.Tasks.CodeAnalysis.dll</RoslynTasksAssembly>
```

Comment:
Really surprised that we don't set `$(CSharpCoreTargetsPath)` or `$(VisualBasicCoreTargetsPath)` here. When we first started using the package via `PackageReference` I _believe_ that we imported a .props from the package which set these values. When we moved to `PackageDownload` did we miss setting these? Can't see anywhere else that these would be set on this code path.